aspnetcore/host-and-deploy/windows-service.md

```diff
@@ -32,23 +32,36 @@ The ASP.NET Core Worker Service template provides a starting point for writing l
 
 [!INCLUDE[](~/includes/worker-template-instructions.md)]
 
----
-
 ::: moniker-end
 
 ## App configuration
 
 ::: moniker range=">= aspnetcore-3.0"
 
-`IHostBuilder.UseWindowsService`, provided by the [Microsoft.Extensions.Hosting.WindowsServices](https://www.nuget.org/packages/Microsoft.Extensions.Hosting.WindowsServices) package, is called when building the host. If the app is running as a Windows Service, the method:
+The app requires a package reference for [Microsoft.AspNetCore.Hosting.WindowsServices](https://www.nuget.org/packages/Microsoft.AspNetCore.Hosting.WindowsServices).
```

Comment:
This is backwards. Ms.Extensions.Hosting.WindowsServices is for generic host 3.0+ and Ms.AspNetCore.Hosting.WindowsServices is for WebHost which is primarily covered in the 2.x docs.